flink-connector-mysql-cdc/src/test/resources/docker/server-gtids/my.cnf

```diff
@@ -36,7 +36,7 @@ skip-host-cache
 skip-name-resolve
 #datadir=/var/lib/mysql
 #socket=/var/lib/mysql/mysql.sock
-#secure-file-priv=/var/lib/mysql-files
```

Comment:
`/var/lib/mysql-files` doesn't  exist in mysql8.0, so change it to `/var/lib/mysql`